shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/screens/NotificationSettings.kt

```diff
@@ -0,0 +1,63 @@
+package org.jetbrains.kotlinconf.screens
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.unit.dp
+import kotlinconfapp.shared.generated.resources.Res
+import kotlinconfapp.shared.generated.resources.notifications_jetbrains_news_description
+import kotlinconfapp.shared.generated.resources.notifications_jetbrains_news_title
+import kotlinconfapp.shared.generated.resources.notifications_kotlinconf_news_description
+import kotlinconfapp.shared.generated.resources.notifications_kotlinconf_news_title
+import kotlinconfapp.shared.generated.resources.notifications_schedule_update_description
+import kotlinconfapp.shared.generated.resources.notifications_schedule_update_title
+import org.jetbrains.compose.resources.stringResource
+import org.jetbrains.kotlinconf.NotificationSettings
+import org.jetbrains.kotlinconf.ui.components.SettingsItem
+
+class NotificationSettingsState {
+    var scheduleUpdates by mutableStateOf(true)
+    var kotlinConfNews by mutableStateOf(true)
+    var jetbrainsNews by mutableStateOf(true)
+
+    val model: NotificationSettings
+        @Composable
+        get() = NotificationSettings(
+            scheduleUpdates = scheduleUpdates,
+            kotlinConfNews = kotlinConfNews,
+            jetbrainsNews = jetbrainsNews,
+        )
+}
```

Comment:
I have mixed feelings about duplicating settings with `mutableStateOf` per each setting. It may slightly improve performance by reducing the number of (re-)compositions, but I doubt it would be noticeable at all.
At the same time, it leads to code duplication and increases complexity.
I'd say passing a `NotificationsSettings` DTO directly would suffice.